fixit/rules/use_assert_is_not_none.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.helpers import ensure_type
+
+from fixit.common.base import CstLintRule
+from fixit.common.utils import InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+class UseAssertIsNotNoneRule(CstLintRule):
+    """
+    Discourages use of ``assertTrue(x is not None)`` as it is deprecated (https://docs.python.org/2/library/unittest.html#deprecated-aliases).
+    Use ``assertIsNotNone(x)`` instead.
+
+    """
+
+    MESSAGE: str = (
+        '"assertTrue" is deprecated. Use "assertIsNotNone" instead.\n'
+        + "See https://docs.python.org/2/library/unittest.html#deprecated-aliases"
```

Comment:
Let's link to Python 3 doc both in this message and the docstring.
https://docs.python.org/3.8/library/unittest.html#deprecated-aliases